docs/02-quick-start/01-installation/02-kubernetes.md

```diff
@@ -1,7 +1,107 @@
 ---
 sidebar_label: Kubernetes
 ---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
 
 # Try Ozone With Kubernetes
 
-**TODO:** File a subtask under [HDDS-9856](https://issues.apache.org/jira/browse/HDDS-9856) and complete this page or section.
+## Minikube
+
+**Requirements:**
+
+* Working minikube setup
+* kubectl
```

Comment:
Please use `-` for unordered lists since it is consistent with the existing pages in the new site. This was supposed to be enforced by markdownlint but I made a mistake in the config. I'll push a fix for the markdownlint config soon.